<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameConfView</class>
 <widget class="QWidget" name="GameConfView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gbChooseGameType">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="rbNewGame">
          <property name="text">
           <string>Create a new game</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbNetworkGame">
          <property name="text">
           <string>Join a network game</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbSetupGame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Setup Game</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="NewGameSetupView" name="phNewGameSetupView"/>
          <widget class="JoinGameSetupView" name="phJoinGameSetupView"/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="gbStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="lblStatus">
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NewGameSetupView</class>
   <extends>QWidget</extends>
   <header>newgamesetupview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JoinGameSetupView</class>
   <extends>QWidget</extends>
   <header>joingamesetupview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rbNewGame</sender>
   <signal>toggled(bool)</signal>
   <receiver>GameConfView</receiver>
   <slot>gameTypeSelectionChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbNetworkGame</sender>
   <signal>toggled(bool)</signal>
   <receiver>GameConfView</receiver>
   <slot>gameTypeSelectionChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gameTypeSelectionChanged(bool)</slot>
 </slots>
</ui>
